internal/app/inspections/inspections.go

```diff
@@ -0,0 +1,133 @@
+package inspections
+
+import (
+	"context"
+	"fmt"
+	"sync"
+
+	"github.com/SafetyCulture/iauditor-exporter/internal/app/api"
+	"github.com/SafetyCulture/iauditor-exporter/internal/app/config"
+	"github.com/SafetyCulture/iauditor-exporter/internal/app/exporter"
+	"github.com/SafetyCulture/iauditor-exporter/internal/app/util"
+	"github.com/spf13/viper"
+	"go.uber.org/zap"
+)
+
+const maxGoRoutines = 10
+
+type tFunc func(api.Inspection)
```

Comment:
What is tFunc?

Reply:
This is used as an argument for throttling the function.
I have used this variable at multiple places, so defined it as a seperate type.
With the current code it is used at only one place, so removed this.